lib/moon/parts/sidebar.ex

```diff
@@ -38,19 +31,16 @@ defmodule Moon.Parts.Sidebar do
       <Drawer.Backdrop class={merge(["lg:hidden", @backdrop_class])} />
       <Drawer.Panel
         position="start"
-        class={merge(["bg-goku shadow-none", "w-[22.5rem]", @panel_class])}
+        class={merge([
+          "h-screen pt-6 pb-5 flex flex-row gap-6",
+          "bg-goku shadow-none w-auto rounded-moon-s-lg",
```

Comment:
I believe, just in case, better to check the rounded corners with Egert? If it has to be rounded or maybe only for mobile view?